service/coordinator_event_subscriber.hh

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2024-present ScyllaDB
+ *
+ * Modified by ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: (AGPL-3.0-or-later and Apache-2.0)
+ */
+
+#pragma once
+
+#include "utils/atomic_vector.hh"
+
+using seastar::future;
+
+namespace service {
+
+class coordinator_event_subscriber {
+public:
+    virtual ~coordinator_event_subscriber() {}
+
+    //TODO: add arguments
+    virtual void on_tablet_migration_start() = 0;
+    virtual void on_tablet_migration_finish() = 0;
+    virtual void on_tablet_migration_abort() = 0;
```

Comment:
what about create/drop table/view?

Reply:
also, do you want to react to topology change events, such as adding/removing a node?